### PR TITLE
[NETSHELL] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR #5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/shellext/netshell/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,5 +1,11 @@
-// COPYRIGHT: Petru Dimitriu <petrimetri@gmail.com>
-//            Ștefan Fulea <stefan.fulea@mail.com>
+/*
+ * PROJECT:     ReactOS Netshell Command
```
Comment (JoachimHenze) [inline dll/shellext/netshell/lang/ro-RO.rc:2]:
```suggestion
 * PROJECT:     ReactOS Netshell
```
This is not a command, but a shell extension.

File: dll/shellext/netshell/lang/ro-RO.rc [hunk 2]
```diff
@@ -139,7 +145,7 @@ BEGIN
     IDS_SHV_COLUMN_TYPE "Tipe"
     IDS_SHV_COLUMN_STATE "Stare"
     IDS_SHV_COLUMN_DEVNAME "Nume dispozitiv"
-    IDS_SHV_COLUMN_PHONE "Tel. sau adresa gazdei"
+    IDS_SHV_COLUMN_PHONE "Număr de telefon sau adresa gazdei"
```
Comment (JoachimHenze) [inline dll/shellext/netshell/lang/ro-RO.rc:148]: This is a column header in a listview. Using such a long string might look weird, as it steals a lot of space from other columns. Please attach a screenshot (at least), and if it looks poor, then please impove on that.

**Comment by JoachimHenze** [comment]
> This is the only thing I can show you as it's a DLL file.

That is not sufficient. Please check it at ros runtime then.

**Comment by AndreiMiloiu** [comment]
I noticed that for Italian, Spanish, French and Portuguese language the language string are even longer.

**Comment by JoachimHenze** [comment]
> I noticed that for Italian, Spanish, French and Portuguese language the language string are even longer.

Please attach a screenshot or a video. Having a nullptr dereference in one module, doesn't make that a valid way to code in a 2nd module neither. Either you can demonstrate your change being fine with a screenshot (or a video operating that dlg), or you shouldn't touch that line.

**Comment by AndreiMiloiu** [comment]
> > I noticed that for Italian, Spanish, French and Portuguese language the language string are even longer.
> 
> Please attach a screenshot or a video. Having a nullptr dereference in one module, doesn't make that a valid way to code in a 2nd module neither. Either you can demonstrate your change being fine with a screenshot (or a video operating that dlg), or you shouldn't touch that line. If it is easier for you then you can also attach a screenshot from Windows that would prove that your string is the same one (or shorter) than what Windows does use. That would be fine as well.

I will revert this change. I don't know at this time how to reproduce the change.

**Comment by JoachimHenze** [comment]
**My expressed concerns have been addressed properly.** I also couldn't spot any accelerator collisions.
I will give others a bit more time to review this PR before expressing my approval, because some of the strings increased in length.

**Comment by AndreiMiloiu** [comment]
> > This is the only thing I can show you as it's a DLL file.
> 
> That is not sufficient. Please check it at ros runtime then.

I reverted the change at this time. I don't know how to do it.

File: dll/shellext/netshell/lang/ro-RO.rc [hunk 3]
```diff
@@ -1,5 +1,11 @@
-// COPYRIGHT: Petru Dimitriu <petrimetri@gmail.com>
-//            Ștefan Fulea <stefan.fulea@mail.com>
+/*
+ * PROJECT:     ReactOS Netshell
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```
Comment (binarymaster) [inline dll/shellext/netshell/lang/ro-RO.rc:3]:
```suggestion
 * PROJECT:    ReactOS Network Shell
 * LICENSE:    LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
```

**Pull request merged**